src/nginxconfig/i18n/en/templates/app.js

```diff
@@ -30,7 +30,7 @@ export default {
     title: `${common.nginx}Config`,
     description: `The easiest way to configure a performant, secure, and stable ${common.nginx} server.`,
     singleColumnMode: 'Single column mode',
-    splitColumnMode: 'Split column mode',
+    splitColumnMode: 'Side by side mode',
```

Comment:
```suggestion
    splitColumnMode: 'Side-by-side mode',
```
'Side-by-side' should be hyphenated, please